extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -405,6 +441,22 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
     const typeConfig = this.predefinedConfig_[type] || {};
 
+    // transport.iframe is only allowed to be specified in typeConfig, not
+    // the others. Allowed when running locally for testing purposes.
+    [defaultConfig, inlineConfig, this.remoteConfig_].forEach(config => {
```

Comment:
Ok, fixed this in extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_3p_analytics.html